index.html

```diff
@@ -281,9 +281,24 @@ <h2>Skills</h2>
             </li> 
           </ul>
       </section>
+
+  
     
 </section>
-
+     <!-- Contact me form  -->
+    <section id="contact" class="contact-me-form">
+        <div class="contact-form">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you
+                 need built or a project that needs coding. I’d love to help with it </p>
+            <form method="post">
```

Comment:
- [ ] Kindly fix this error after submitting i get this message
![Capturef](https://user-images.githubusercontent.com/81444338/171379845-2b0540a6-485a-43b8-98fd-8877487ec471.PNG)
- [ ] Kindly follow the requirement you forgetten to add action to the form  see example _**action value should be similar to https://formspree.io/f/{form_id}**_